packages/apps/frontera/src/routes/organization/src/components/Tabs/panels/AccountPanel/Contract/ContractBillingDetailsModal/InlineSelect.tsx

```diff
@@ -34,6 +34,7 @@ export const InlineSelect: FC<InlineSelectProps> = ({
       placeholder={placeholder}
       name={name}
       formId={formId}
+      defaultMenuIsOpen={true}
```

Comment:
Is this intentional or leftover from development?

Reply:
:D leftover